js/app/modules/card/knowledgeDocument.js

```diff
@@ -237,7 +247,7 @@ const KnowledgeDocument = new Module.Class({
     _get_webview: function () {
         let webview = this._create_webview();
 
-        webview.renderer.enable_scroll_manager = this.show_toc;
+        webview.renderer.enable_scroll_manager = true;
```

Comment:
This approach uses our JS [scroll manager](https://github.com/endlessm/eos-knowledge-lib/blob/master/data/templates/js/scroll-manager.js#L33) helper (same as when you click on the TOC). Before, made no sense to have it enabled if the TOC wasn't display, but now it does, because you want this to work even if the TOC is not displayed.